frontend/ui/size.lua

```diff
@@ -26,7 +26,18 @@ defaults, please take a second to consider:
 ]]
 
 local dbg = require("dbg")
-local Screen = require("device").screen
+local Device = require("device")
+local Screen = Device.screen
+
+-- This function calculates reasonable large number of items based on screen properties
+-- px is the screen_height or screen_width
+-- 72 is used because there are 72 points per inch (in printing press)
+-- limit is points (pt), i.e., what is minimum number of points to make this item readable
+-- Items might be number of lines in the list (pt = 20), of mosaic grid cover size (pt = 75)
+function SizeMaxItemsNorm(px, dpi, limit)
```

Comment:
`local function`

Reply:
(Function name would also need to be snake_case to follow our style guide)

Reply:
Thanks! I tried to simplify it further. Let's see if this is better.

Also, I noticed that I didn't require `math`, but it doesn't complain either.

Reply:
`math` is a builtin module (on luajit at least).

Reply:
> (Function name would also need to be snake_case to follow our style guide)

Regular functions lowerCamelCase. ^_^

Reply:
> (Function name would also need to be snake_case to follow our style guide)

Really ? :) I think they ought to be camelCase.

Reply:
I had a vague memory it said something along the lines of: function => snake_case vs. methods => camelCase

(I haven't double-checked ^^).